pysindy/optimizers/constrained_sr3.py

```diff
@@ -188,6 +199,23 @@ def __init__(
             self.unbias = False
             self.constraint_order = constraint_order
 
+        if inequality_constraints and not self.use_constraints:
+            raise ValueError(
+                "Use of inequality constraints requires constraint_lhs and "
+                "constraint_rhs."
+            )
+
+        if inequality_constraints and (
+            thresholder.lower() != "l1"
+            and thresholder.lower() != "l2"
+            and thresholder.lower() != "weighted_l1"
+            and thresholder.lower() != "weighted_l2"
```

Comment:
There are options for "l2" and "weighted_l2" in here, but those are not implemented for SR3? Is this an inconsistency?

Reply:
They are implemented for SR3 now! Although they are kind of dumb, since ridge regression typically works best with a l1 or l0 norm in addition to the L2.